packages/core/src/events/zone/root-task-invoked-event.class.ts

```diff
@@ -0,0 +1,8 @@
+import { Probe } from '../../probes';
+import { TaskEventSupport } from './task-event-support.class';
+
+export class RootTaskInvokedEvent extends TaskEventSupport {
+  constructor(probe: Probe, task: any) {
```

Comment:
Can task be type aliased. Is this a ZoneTask or Macro/MicroTask, etc.?

Reply:
It's a ZoneTask, which we don't have the type for :(

Reply:
I can add an alias